samples/sample_encode/src/pipeline_encode.cpp

```diff
@@ -1663,6 +1719,61 @@ mfxStatus CEncodingPipeline::ResetMFXComponents(sInputParams* pParams)
     return MFX_ERR_NONE;
 }
 
+mfxStatus CEncodingPipeline::AllocExtBuffers(sInputParams *pInParams)
```

Comment:
you have to check pointer, or use reference instead

Reply:
OK